### PR TITLE
initial success

### PR DESCRIPTION
**Pull request by prima101112**

### CODE REVIEW

File: files/config/config.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[General]
+Port = 6379
+SetToDestWhenGet = on
```
Comment (ekarisky) [inline files/config/config.ini]: can you make this boolean value?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	redis "github.com/prima101112/go-redis-server"
```
Comment (ekarisky) [inline main.go]: are you sure using this? not docker/go-redis-server?

**Review by ekarisky — CHANGES_REQUESTED** [review]
please add unit test also @prima101112

File: files/config/config.ini (hunk 1 above)
Comment (prima101112) [reply files/config/config.ini]: its already boolean . . . ther role for boolean is on and off

File: main.go (hunk 2 above)
Comment (prima101112) [reply main.go]: i'm sure to lock version 
but ist up to my leader ✌️

File: files/config/config.ini (hunk 1 above)
Comment (prima101112) [reply files/config/config.ini]: @ekarisky

Comment (ekarisky) [reply files/config/config.ini]: i mean true or false

File: main.go (hunk 2 above)
Comment (ekarisky) [reply main.go]: use this github.com/tokopedia/go-redis-server 👍

**Review by ekarisky — APPROVED** [review]

**Pull request merged**